documentation/more.php

```diff
@@ -208,4 +208,4 @@
 		</div>
 	</div>
 
-</section>
\ No newline at end of file
+</section>
```

Comment:
It's not about the new line at the end, right? :-) How about another line change?